ImageEditor/build.gradle

```diff
@@ -27,30 +29,40 @@ android {
 
     sourceSets {
         main.java.srcDirs += 'src/main/kotlin'
-        release.java.srcDirs += 'src/release/kotlin'
-        debug.java.srcDirs += 'src/debug/kotlin'
         test.java.srcDirs += 'src/test/kotlin'
         androidTest.java.srcDirs += 'src/androidTest/kotlin'
     }
 }
 
 dependencies {
-    def nav_version = "2.0.0"
+    def navComponentVersion = "2.0.0"
 
     implementation fileTree(dir: 'libs', include: ['*.jar'])
+    // Kotlin
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'androidx.appcompat:appcompat:1.0.2'
 
-    implementation 'com.github.bumptech.glide:glide:4.10.0'
-    implementation ('com.github.yalantis:ucrop:2.2.2') {
-        exclude group: 'com.squareup.okhttp3'
-    }
+    // Dagger
```

Comment:
If we ever plan to move it to another repo and use it as a library I think adding unnecessary external dependencies, such as Dagger, might discourage some potential users. Wdyt?

Reply:
Removed here af5488a